### ISSUE TITLE
Use realtime recognition while other applications are recording

### ISSUE THREAD

**Issue by stefan-reich** [Picovoice/cheetah#30]
I'm opening this as a new issue because I don't have a solution. I need to record in parallel with JavaSound and Picovoice on Linux/Raspbian. USB soundcards are used. Thanks.

**Comment by kenarsa** [comment]
the GitHub version only runs on x86_64 Linux, not arm. the integration is up to you and not an issue related to the engine.
